src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapper.java

```diff
@@ -69,7 +61,7 @@
  */
 public abstract class KNNVectorFieldMapper extends ParametrizedFieldMapper {
 
-    private static Logger logger = LogManager.getLogger(KNNVectorFieldMapper.class);
+    static Logger logger = LogManager.getLogger(KNNVectorFieldMapper.class);
```

Comment:
realized that lombok logger is only private. we used that old logger in one of the sub-classes, I guess that's ok to create it's own logger specifically for that class. It will change a log file content a bit, as I think it places class name in the line prefix.

Reply:
make it protected rather than package private, why not create the logger in the class which is using it rather than creating it in the parent class.